docs/standard/frameworks.md

```diff
@@ -1,56 +1,293 @@
 
-## Framework Versions
+Refernece | Name
+--- | ---
+Product | .NET Framework 4.6.2 or .NET 4.6.2
+Framework | `.NETFramework,Version=4.6.2` or `net462`
+Family | `.NETFramework` or `net`
 
-The table below defines the set of frameworks that you can use, how they are referred to and which version of the [.NET Standard Library](library.md) that they implement.
+## Lastest framework versions
 
-| Framework | Latest Version | Target Framework Moniker (TFM) | Compact Target Framework Moniker (TFM) | .NET Standard Version | Metapackage |
-|:--------: | :--: | :--: | :--: | :--: | :--: | :--: |
-| .NET Standard | 1.6 | .NETStandard,Version=1.6 | netstandard1.6 | N/A | [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library)|
-| .NET Core Application | 1.0.1 | .NETCoreApp,Version=1.0 | netcoreapp1.0 | 1.6 | [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App)|
-| .NET Framework | 4.6.2 | .NETFramework,Version=4.6.2 | net462 | 1.5 | N/A |
+The table below defines the set of frameworks that you can use, how they're referenced, and which version of the [.NET Standard Library](library.md) they implement. These framework versions are the latest stable versions. Pre-release versions aren't shown.
 
-> [!NOTE]
-> These framework versions are the latest stable versions. There may be pre-released versions as well that are not described by this table.
+Framework | Latest Version | Target Framework Moniker (TFM) | Compact Target Framework Moniker (TFM) | .NET Standard Version | Metapackage
+:--------: | :--: | :--: | :--: | :--: | :--: | :--:
+.NET Standard | 1.6.1 | .NETStandard,Version=1.6 | netstandard1.6 | N/A | [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library)
+.NET Core Application | 1.1.1 | .NETCoreApp,Version=1.1 | netcoreapp1.1 | 1.6 | [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App)
+.NET Framework | 4.6.2 | .NETFramework,Version=4.6.2 | net462 | 1.5 | N/A
 
-## Writing about Frameworks
+## Supported frameworks
 
-There are multiple ways to refer to frameworks in written form, most of which are used in this documentation. They are described below, both as a legend for interpreting the documentation but also to guide use in other documents.
+A framework is typically referenced by a short target framework moniker or *TFM*. In .NET Standard, this is also generalized to *TxM* to allow a single reference to multiple frameworks. The NuGet clients support the following frameworks. Equivalents are shown within brackets (`[]`).
 
-Using .NET Framework 4.6.1 as an example, the following forms can be used:
+Name | Abbreviation | TFMs/TxMs
+--- | --- | ---
+.NET Framework | net | net11
+ | | net20
+ | | net35
+ | | net40
+ | | net403
+ | | net45
+ | | net451
+ | | net452
+ | | net46
+ | | net461
+ | | net462
+.NET Core | netcore | netcore [netcore45]
```

Comment:
Depreciated .NET Core TFM's? ... that haven't been formally moved to the "depreciated" list yet because some libs perhaps still use them??? *wilding guessing*